backend/src/main/backend/Misc.txt

```diff
@@ -0,0 +1,32 @@
+Problems:
+--> [FIXED?]
+    the table class is too general. each instance of the class can create
```

Comment:
Finding the right balance is tough. Here's my suggestion:
1. Combine only what is truly shared between the different tables. If it's not exactly the same, or nearly the same, it doesn't need abstracted.
2. Embrace what's different about the different tables. Don't force method signatures to fit all cases purely for the sake of abstraction. This is tough to get right and tends to evolve over time.